### ISSUE TITLE
Support aws route tables 

### ISSUE THREAD

**Issue by szuecs** [netenglabs/suzieq#311]
Thanks for the great visibility software!

I would like to have AWS route tables integrated. There are also other network components like natgateway or transitgateway which would be interesting.
I am not very familiar with transitgateway, but the others are just static and you can use aws sdk to read via APIs all the information.

**Comment by ddutt** [comment]
Thanks for engaging. We'll add this to our roadmap. Is there a timeline that you're aiming to try to use this functionality?

**Comment by szuecs** [comment]
No, I don't have a timeline. Whenever you think it makes sense to add. Thanks for the response!
